### DISCUSSION TITLE
CSS not properly loaded in GitHub Pages but works locally..

### DISCUSSION THREAD

**Discussion by tylertraviss** [community/community#27974] [Other Feature Feedback, Questions, & Ideas] [answered]
After developing a portfolio, it seems that the CSS is not configured when uploading to GitHub Pages. The GitHub pages being loaded is: [my website](https://tylertraviss.github.io/WebsitePortfolio/), and the repository is [here](https://github.com/tylertraviss/WebsitePortfolio). After troubleshooting, I still can't identify the issue. Any advice is lovely! Thanks.

**Comment by jfullstackdev** [comment, marked as answer]
this is what I'm seeing on the live website

![image](https://user-images.githubusercontent.com/47092464/182287494-3dc4d0f9-d54f-4a96-8a7f-d4e122f5b971.png)

but there are so many resources not being loaded 

you need to update these links 

![image](https://user-images.githubusercontent.com/47092464/182287579-49f9b314-c8e8-4751-befd-f2963e17516f.png)

as you can see if I follow the URL, this will generate, 

<https://tylertraviss.github.io/assets/img/favicon/apple-icon-57x57.png>

which is not correct, it should be 

![image](https://user-images.githubusercontent.com/47092464/182287726-10a7c0c7-c0a5-486e-b4a2-59ab9b57a367.png)

<https://tylertraviss.github.io/WebsitePortfolio/assets/img/favicon/apple-icon-57x57.png>

just simply add the repo name which is the missing one `WebsitePortfolio`

**Reply by tylertraviss** [reply]
Thanks so much for your help, the website is deployed properly now. All the best!
